addons/binding/org.openhab.binding.yamahareceiver.test/.classpath

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
+	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
+	<classpathentry kind="src" path="src/test/java"/>
```

Comment:
The tests also work in Eclipse by adding the resources directory to the classpath. :smile: 

```xml
	<classpathentry kind="src" path="src/test/resources"/>
```

Reply:
Added this to my commit. 

This file came from the test project maven archetype, maybe worth including this in the archetype?

Reply:
Neither the binding or the binding.test archetypes have a resources directory by default. So that's the reason why it's not added in the archetypes.

Perhaps we could add a SAT check to verify that resource directories are not missing from .classpath files @svilenvul? That way people are triggered to add them to .classpath files even if they don't use Eclipse themselves for development.

There might be other such issues also for people who don't use IntelliJ. :wink: